### ISSUE TITLE
Self-checkout page, footer not showing

### ISSUE THREAD

**Issue by mazgor** [tobiaszimmermann1/foodcoop-plugin#31]
The footer on the Foodcoop self-checkout page is not displayed if, for example, the handheld navigation bar is not hidden with the Woocommerce Storefront theme. It is also not visible when scrolling. I used/found the following code in order to hide the handheld navigation bar:

add_action( 'init', 'jk_remove_storefront_handheld_footer_bar' );

function jk_remove_storefront_handheld_footer_bar() {
  remove_action( 'storefront_footer', 'storefront_handheld_footer_bar', 999 );
}

Like this it works and the foodcoop footer is visible. Maybe this code could be integrated in the shortcode [foodcoop_addtocart] so that the theme's footer is always disabled, but only on that specific page?

![IMG_6730](https://github.com/tobiaszimmermann1/foodcoop-plugin/assets/35103175/8e37ae47-e328-4291-9001-bc5bb570c305)

**Comment by tobiaszimmermann1** [comment]
i don't think that it makes sense to integrate theme specific actions into the plugin. I think a more suitable approach would be to disable the themes css and set the self checkout div to a high z-index
